spec/tbd_tests_spec.rb

```diff
@@ -62,14 +62,90 @@
 
     # Valid JSON entries.
     TBD.clean!
-    
+    version = OpenStudio.openStudioVersion.split(".").join.to_i
 
+    # The v1.11.5 (2016) seb.osm, shipped with OpenStudio, holds (what would now
+    # be considered as deprecated) a definition of plenum floors (i.e. ceiling
+    # tiles) generating several warnings with more recent OpenStudio versions.
```

Comment:
Harmonized with (more detailed) [fixes](https://github.com/rd2/tbd_tests/pull/32) introduced in TBD Tests.